salt/utils/cloud.py

```diff
@@ -405,10 +405,7 @@ def bootstrap(vm_, opts):
         'tmp_dir': salt.config.get_cloud_config_value(
             'tmp_dir', vm_, opts, default='/tmp/.saltcloud'
         ),
-        'deploy_command': salt.config.get_cloud_config_value(
```

Comment:
As it happens, this parameter was  stepped on in the implementation, so anything passed would be useless.  Under the principle of least surprise, a parameter which looks like it should work, but does not, seems to be a bad idea.  The value of `deploy_command` is built from a config.get call at about line 1246.  (I would not have noticed this -- PyCharm actually caught the error.)